packages/tradershub/src/components/Modal/ModalContent.tsx

```diff
@@ -10,7 +10,7 @@ import { TModalComponents } from './Modal';
  * @returns {JSX.Element} The ModalContent component.
  */
 const ModalContent = ({ children, className }: TModalComponents) => (
-    <div className={qtMerge('flex-grow overflow-y-auto p-400 lg:flex-none', className)}>{children}</div>
+    <div className={qtMerge('flex-grow overflow-y-auto p-50 sm:p-400 lg:flex-none', className)}>{children}</div>
```

Comment:
We are going with mobile first approach
cc: @aizad-deriv 
```suggestion
    <div className={qtMerge('flex-grow overflow-y-auto lg:p-50 p-400 lg:flex-none', className)}>{children}</div>
```

Reply:
We are going with mobile first approach and sm:p-400 means that apply padding 400 to screens above mobile, we don't want padding on mobile screens.